client/my-sites/plans/current-plan/index.jsx

```diff
@@ -49,7 +50,8 @@ class CurrentPlan extends Component {
 		isExpiring: PropTypes.bool,
 		shouldShowDomainWarnings: PropTypes.bool,
 		hasDomainsLoaded: PropTypes.bool,
-		isAutomatedTransfer: PropTypes.bool,
+		showJetpackChecklist: PropTypes.bool,
```

Comment:
Not necessarily in this PR: With a large list of prop types like this one, I think it's essential to separate connected props from the others by a dedicated comment.